apache-spark-worker/build/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu
+
+MAINTAINER Frank Austin Nothaft, fnothaft@berkeley.edu
+
+RUN apt-get update && \
+  apt-get install -y \
+    python \
+    libnss3 \
+    openjdk-8-jdk \
+    curl
+
+COPY download.sh /home/
+RUN sh /home/download.sh
+
+WORKDIR /opt/apache-spark
+RUN sed -i "s/<joda.version>2.5<\/joda.version>/<joda.version>2.9.4<\/joda.version>/" pom.xml
```

Comment:
Might be safer to [`git apply`](https://git-scm.com/docs/git-apply) the patch, and to then do a `test -z "git status --porcelain"`